.github/workflows/tests.yml

```
@@ -24,11 +24,10 @@ jobs:
         python -m pip install --upgrade pip
         pip install .
         
-    - name: Run plugin tests
+    - name: Run gltest cli tests
       run: |
-        gltest tests/plugin
+        pytest tests/gltest_cli/
     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)